Task_functions/index.js

```diff
@@ -1,11 +1,13 @@
-//Write code for the 1st task below this
-
```

Comment:
Consider using concise arrow function syntax without curly braces and return keyword for better readability.